### PR TITLE
[Feature][POC] close

### PR DESCRIPTION
**Pull request by caneGuy**
## What type of PR is this：
- [ ] bug
- [x] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

### CODE REVIEW

**Comment by caneGuy** [comment]
@decster @sevev could you help review this poc pr? thanks

**Comment by caneGuy** [comment]
Just use fix column name and do not consider performance issue

File: be/src/storage/tablet_updates.cpp [hunk 1]
```diff
@@ -827,8 +835,61 @@ void TabletUpdates::_apply_rowset_commit(const EditVersionInfo& version_info) {
...
+                std::vector<uint64_t> old_rowids(upserts[i]->size());
+                index.get(*upserts[i], &old_rowids);
+                std::map<uint32_t, std::vector<uint32_t>> old_rowids_by_rssid;
+                size_t num_default = 0;
+                vector<uint32_t> idxes;
+                RowsetUpdateState::plan_read_by_rssid(old_rowids, &num_default, &old_rowids_by_rssid, &idxes);
+                std::vector<std::unique_ptr<vectorized::Column>> old_columns(1);
+                auto old_column = ChunkHelper::column_from_field_type(tablet_column.type(), tablet_column.is_nullable());
+                old_columns[0] = old_column->clone_empty();
+                get_column_values(read_column_ids, false, old_rowids_by_rssid, &old_columns);
```
Comment (sevev) [inline be/src/storage/tablet_updates.cpp:852]: After call this function, the data order of `old_columns` is not match with `keys`, i think you need to reorder the data according to `idxes`.

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 4 / 5 (80.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/load/routineload/RoutineLoadJob.java|1|2|50.00%|[292]|
|:large_blue_circle:|com/starrocks/analysis/CreateRoutineLoadStmt.java|3|3|100.00%|[]|

**Pull request closed without merging**